SourcingTests/UITableCollectionViewBaseMock.swift

```diff
@@ -28,7 +28,7 @@
 
 import UIKit
 
-// swiftlint:disable force_cast force_try force_unwrapping
```

Comment:
You can delete this file. I think it is never used

Reply:
ok